juju/client/connection.py

```diff
@@ -642,7 +645,7 @@ async def rpc(self, msg, encoder=None):
         if "version" not in msg:
             msg['version'] = self.facades[msg['type']]
         outgoing = json.dumps(msg, indent=2, cls=encoder)
-        log.debug('connection {} -> {}'.format(id(self), outgoing))
+        log.debug('connection id: {} -- sending {}'.format(id(self), outgoing))
```

Comment:
I actually prefer the "->" vs "<-" to 'sending' and 'receiving' because it means the lines line up in debug output and the arrows aren't too hard to interpret.
we could go with 'send' and 'recv', I suppose

Reply:
Yeah, I think the `->` and `<-` *looks* better, but I got frustrated when I was manually reading the debug output that I was misinterpreting some of them, that's why I thought the change might make it clearer. But I haven't considered that the lines line up in the output, so *that* might be a good reason to revert this 👍

Reply:
I think I'll make them `--->` and `<---`, which might make it harder to misinterpret, and would still line up